internal/applicationsnapshot/report.go

```diff
@@ -159,6 +160,7 @@ func NewReport(snapshot string, components []Component, policy policy.Policy, da
 		Data:          data,
 		PolicyInput:   policyInput,
 		EffectiveTime: policy.EffectiveTime().UTC(),
+		ShowSuccesses: showSuccesses,
```

Comment:
As discussed, should also have this one based on the --info flag:

```diff
+                      ShowInfo: showInfo,
```

Could be done later.

Reply:
Looking at the code that might be a bit more work, so I'll file a separate story about this